### ISSUE TITLE
Fix sending new string notifications

### ISSUE THREAD

**Issue by bugzilla-to-github** [mozilla/pontoon#2063]
*This issue was created automatically by a [script](https://github.com/mathjazz/bugzilla2github/).*
## [Bug 1359033](https://bugzilla.mozilla.org/show_bug.cgi?id=1359033)
Bug Reporter: @mathjazz
CC:   @MikkCZ
Blocker for:  [Bug 1379158](https://bugzilla.mozilla.org/show_bug.cgi?id=1379158)

There are a few cases when we send new strings notifications but we shouldn't, and vice versa.

We do not send notifications, but we should:

- If new symmetric (bilingual) file is enabled for a locale (e.g. in Mozilla.org), we should notify contibutors of that localization about new strings. Currently we don't send any notifications when such event occurs.

We do send notifications, but we should not:

- If we not only add new strings, but also pretranslate them in repository using existing translations (e.g. on merge to Firefox Beta or when files are moved in the l10n repo on merge day), we shouldn't notify project contributors about them.

- If source strings are different depending on locale (e.g. Mozilla.org or Engagement), we should send notifications only to contibutors of locales for which the new strings were added. Currently we notify all project contributors, even if new strings are only enabled for a handful of locales.

- If a project is private, we send new string notifications to all translation authors within that project. That doesn't sound wrong on a first look, but if the project had been public in the past and is not private, there might be false positives among the recipients. So we must only send notifications to users that have access to the project at the time of sending a notification.

**Comment by flodolo** [comment]
> * If we not only add new strings, but also pretranslate them in repository using existing translations (e.g. on merge to Firefox Beta or when files are moved in the l10n repo on merge day), we shouldn't notify project contributors about them.

I disagree that we should stop sending notifications in this case. We know people want to know when new strings are added to their project, even if pretranslated. These days, the only case for that would be Fluent migrations.
